src/tiles/list/tile-list-compact.ts

```diff
@@ -24,6 +24,8 @@ export class TileListCompact extends LitElement {
 
   @property({ type: Object }) sortParam: SortParam | null = null;
 
+  @property({ type: String }) selectedCreatorFilter?: string;
```

Comment:
Selected feels a bit redundant; `createrFilter` or `creatorLetterFilter` seem a little clearer to me.

```suggestion
  @property({ type: String }) creatorFilter?: string;
```